src/components/Nav.astro

```diff
@@ -22,7 +22,7 @@ const { links, class: className } = Astro.props;
       ))
     }
   </ul>
-  <NavHamburger class="hamburger" links={links} />
+  <NavHamburger class="nav-hamburger" links={links} />
```

Comment:
Added because there was a clash between class names in this component and `<NavHamburger />`.

This issue only occured in local dev and appears related to [this astro issue](https://github.com/withastro/astro/issues/6975).